src/pages/api/execute/tasks.ts

```diff
@@ -14,7 +14,7 @@ export default async (req: NextApiRequest, res: NextApiResponse) => {
     logger.debug("Request details:", { method: req.method, url: req.url });
 
     const session = await getServerSession(req, res, authOptions);
-    if (!session && env.NODE_ENV === "production") {
+    if (!session) {
```

Comment:
@abhishekkundu25 You asked me to remove this while we were troubleshooting the access-issues. Shoud we keep it like this or has this been commited wrongfully?

Reply:
yes we can keep it like this, env.NODE_ENV === "production" is a bit problamatic for deployment and build, removed from other places as well